js_modules/dagit/packages/ui/src/components/Tooltip.tsx

```diff
@@ -28,17 +28,21 @@ const overwriteMerge = (destination: any[], source: any[]) => source;
 interface Props extends Tooltip2Props {
   display?: React.CSSProperties['display'];
   canShow?: boolean;
+  useDisabledButtonTooltipFix?: boolean;
 }
 
 export const Tooltip: React.FC<Props> = (props) => {
-  const {children, display, canShow = true, ...rest} = props;
+  const {useDisabledButtonTooltipFix = false, children, display, canShow = true, ...rest} = props;
+
+  const [isOpen, setIsOpen] = React.useState<undefined | boolean>(undefined);
 
   if (!canShow) {
     return <>{children}</>;
   }
 
-  return (
+  const styledTooltip = (
     <StyledTooltip
+      isOpen={isOpen}
       {...rest}
       minimal
       $display={display}
```

Comment:
I think this prop probably needs to be applied to the wrapper div!

Reply:
`isOpen` gets passed to blueprint's Tooltip component. It switches the Tooltip to be a controlled rather than uncontrolled component

Reply:
oh you meant `$display`?